backend/backend.go

```diff
@@ -313,22 +313,26 @@ func (backend *Backend) createAndAddAccount(
 		if err != nil {
```

Comment:
The comment is for the line above this. I'm always confused about getSigningConfigurations. Can it be a normal arg with type `signing.Configurations`?

Reply:
This is coming in one of the next PRs. Here is the sneak peak of the commit message:

----
backend/accounts: load signing configurations immediately

The reason that the signing configurations were loaded lazily was that
it requires querying the xpubs from the device. With the BitBox01,
this is especially slow, but also the BitBox02 shows a small delay
when loading e.g. 3 xpubs.

Since we want to persist the accounts/configs anyway (to support
multiple accounts and watch-only), we need to get them
immediately. Since they will be persisted, this will only happens once
per keystore.